### PR TITLE
Fennecs.

### PR DESCRIPTION
**Pull request by george99g**
### Intent of your Pull Request

Fennecs. Stuff. Last one didn't have that one file.

@Kmc2000

### CODE REVIEW

**Comment by Kmc2000** [comment]
Adds some cute fennec foxes

Was meant to be a test to try get git working, but i'm bad

**Comment by george99g** [comment]
@Kmc2000 Screenshots would be nice.

**Comment by Kmc2000** [comment]
https://gyazo.com/73093c0fe597305adab397b4838cf48e

**Comment by Cruix** [comment]
If it's a fox then why is it not a subtype of fox?

**Comment by AsV9** [comment]
^^^^

File: yogstation.dme [hunk 1]
```diff
@@ -653,6 +653,7 @@
 #include "code\game\objects\items\stacks\sheets\mineral.dm"
 #include "code\game\objects\items\stacks\sheets\sheet_types.dm"
 #include "code\game\objects\items\stacks\sheets\sheets.dm"
+
```
Comment (AsV9) [inline yogstation.dme:658]: _nooooooooo_
not the addition line for no reason
noooo

**Comment by Kmc2000** [comment]
ERMMMM

Because reasons

**Comment by ShadowDeath6** [comment]
Why does it has multiple frames per icon state, no other pet other than poly does this and poly only does it sometimes
Fucking triggered

**Comment by Notamaniac** [comment]
Regarding the sprite

A : If you wanted the animate the sprite, just have it bounce in place rather than walking, right now it looks like it's legs are vibrating, especially with the delay set to 1. You've also got to consider that the mob itself won't be moving at times, so it'll look odd when it's just standing still.

B : You've used pure black for the eyes, nose and paws, which stands out a lot. I'd use a very dark yellow shade/brown to avoid that, but then again personal preference.

can-you-keep-a-fennec-fox-in-the-uk-as-a-pet-53cf959ca2ed9

C : Looking at a real life fennec, their heads seem to be a bit smaller with more pronounced ears. I'd reduce the radius of the head by a pixel, and draw those ears a pixel or two longer

**Comment by Kmc2000** [comment]
Pseudo, it's a movement state

It only happens when it moves

**Comment by TheSadFox** [comment]
Not a playable race.
1f44e

**Comment by DavidEdwards** [comment]
> If it's a fox then why is it not a subtype of fox?

**Comment by Kmc2000** [comment]
BECAUSE REASONS THAT'S WHY

**Comment by X-TheDark** [comment]
No, seriously. Why is a fox-type thing not a subtype of fox? I'm voting **Against** unless the question is answered.

**Comment by Cruix** [comment]
THAT WAS A MISCLICK I AM NOT ASSIGNING MYSELF TO IT

**Comment by Notamaniac** [comment]
Right, missed the giant M there. Still I don't think the movement is going to look natural when in motion.

**Comment by Super3222** [comment]
Make fennecs purchasable with cargo! So Renault can have a friend, if the cargo techs are willing.

**Comment by Kmc2000** [comment]
Guys can we merge this ASAP it's a critical feature

**Comment by ShadowDeath6** [comment]
Not a playable race. :+1:

**Comment by AsV9** [comment]
this is not plonked down on the map yet so if it looks horrible we can go ahead and fix it without any tears felt

**Pull request merged**